### PR TITLE
Pull request commit list

### PR DESCRIPTION
**Pull request by eanker**
image

### CODE REVIEW

File: src/pull-request/pull-request-commits.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      .top-container {
+        display: flex;
+        flex-direction: row;
+        margin-bottom: 2px;
+      }
+      .container {
+        display: flex;
+        flex-direction: column;
+      }
+      h4 {
```
Comment (TimvdLippe) [inline src/pull-request/pull-request-commits.html]: This selector is unused.

File: src/util/toggle-content.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <iron-icon icon="icons:arrow-downward" class$="toggle-comments-icon [[showContent]]" on-tap="_toggleContent"></iron-icon>
+    <span hidden$="[[showContent]]">
+      <content></content>
+    </span>
+  </template>
+  <script>
+    Polymer({
+      is: 'toggle-content',
+      properties: {
+        showContent: {
```
Comment (TimvdLippe) [inline src/util/toggle-content.html]: I think it is clearer to name this `hideContent`, as `true` hides it and `false` shows it.

**Comment by eanker** [comment]
There seems to be an issue that when clicked on one show more, that happens at all commits.
I can't seem to figure out why.

**Comment by TimvdLippe** [comment]
LGTM. If we can extract an issue for the incorrect collapsing, we can merge this as-is.

**Pull request merged**